src/palindrom-dom.js

```diff
@@ -236,24 +211,9 @@ const PalindromDOM = (() => {
     }
   }
 
-  PalindromDOM.prototype = Object.create(Palindrom.prototype);
-
-  /* backward compatibility, not sure if this is good practice */
-  if (typeof global === 'undefined') {
-    if (typeof window !== 'undefined') {
-      /* incase neither window nor global existed, e.g React Native */
-      var global = window;
-    } else {
-      var global = {};
-    }
-  }
```

Comment:
I thought the above code was added for compat with Node and React Native (https://github.com/Palindrom/Palindrom/commit/d84db53b60bcc7b5b2f01799a0f80754d01764e0) and has nothing to do with PuppetJS.

Reply:
Semi-true. It was there to have the correct global under different environments. But only to use to inject PuppetJs global for backward compat.